bot.py

```
@@ -203,6 +210,44 @@ async def feedback(message: Message):
+@router.message(F.voice)
+async def voice_to_text(message: types.Message, bot: Bot):
+    """
+    Обрабатывает голосовые сообщения, расшифровывает их и отправляет текст обратно.
+    """
+    try:
+        # Скачиваем голосовое сообщение
+        voice_file = await bot.get_file(message.voice.file_id)
+        ogg_path = f"voice_{voice_file.file_id}.ogg"
+        download = await message.reply('Скачиваю сообщение')
+        await bot.download_file(voice_file.file_path, ogg_path)
+        # Конвертируем из OGG в WAV, так как SpeechRecognition лучше работает с WAV
```

Ruff (0.12.2)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)